### ISSUE TITLE
Remove twisted.internet.qtreactor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8288]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8288|
|Type|task|
|Created|2016-04-12 11:00:26Z|
|Branch|https://github.com/twisted/twisted/tree/no-qtreactor-8288|

It's been outside of Twisted long enough, and there's more modern replacements like https://pypi.python.org/pypi/qt5reactor . We should remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__8288 8288
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_no_qtreactor_8288 branches/no-qtreactor-8288
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1460458826276405 1460458826276405
changetime__1460684573387336 1460684573387336
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [47219]) Branching to no-qtreactor-8288.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Builders spun, I'm happy with this.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Lood good. thanks! please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [47239]) Merge no-qtreactor-8288: Remove deprecated twisted.internet.qtreactor

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8288](https://github.com/twisted/twisted/issues/8288)
